src/schema/geneInteraction.yaml

```diff
@@ -0,0 +1,467 @@
+      - BFO:0000001
+      - WIKIDATA:Q35120
+      # UMLS Semantic Group "Objects"
+      - UMLSSG:OBJC
+      # Entity
+      - UMLSSC:T071
+      - UMLSST:enty
+
+  BiologicalEntity:                         # this should be generalized in core.yaml
+    is_a: NamedThing
+    abstract: true
+    exact_mappings:
+      - biolink:biological_entity
+    narrow_mappings:
+      - WIKIDATA:Q28845870
+      # UMLS Semantic Type "Experimental Model of Disease"
+      - UMLSSC:T050
+      - UMLSST:emod
+      # SIO term is 'biological entity' but less inclusive than the Biolink scope
+      - SIO:010046
+
+  MolecularEntity:                         # this should be generalized in core.yaml
+    is_a: BiologicalEntity
+    description: >-
+      A gene, gene product, small molecule or macromolecule (including protein complex)"
+    exact_mappings:
+      - CHEBI:24431
+      - biolink:molecular_entity
+
+  ## ------------
+  ## ASSOCIATIONS
+  ## ------------
+
+  Association:                     # this should be generalized in core.yaml
+    is_a: Entity
+    description: >-
+      A typed association between two entities, supported by evidence
+    comments:
+      - This is roughly the model used by biolink and ontobio at the moment
+    slots:
+      - subject
+      - predicate
+      - object
+      - relation
+      - negated
+      - qualifiers
+      - association_publications
+    slot_usage:
+      type:
+        description: rdf:type of biolink:Association should be fixed at rdf:Statement
+      category:
+        range: category type
+        required: false
+    exact_mappings:
+      - OBAN:association
+      - rdf:Statement
+      - owl:Axiom
+      - biolink:association
+
+  GeneToGeneAssociation:
+    description: >-
+      abstract parent class for different kinds of gene-gene or gene product
+      to gene product relationships. Includes homology and interaction.
+    abstract: true
+    is_a: Association
+    defining_slots:
+      - subject
+      - object
+    slot_usage:
+      subject:
+        range: Gene
+        description: >-
+          the subject gene in the association. If the relation is symmetric,
+          subject vs object is arbitrary. We allow a gene product to stand
+          as a proxy for the gene or vice versa.
+      object:
+        range: Gene
+        description: >-
+          the object gene in the association. If the relation is symmetric,
+          subject vs object is arbitrary. We allow a gene product to stand
+          as a proxy for the gene or vice versa.
+    exact_mappings:
+      - biolink:gene_to_gene_association
+
+  GeneInteraction:
+    description: >-
+      An interaction between two genes or gene products; this may be a physical
+      molecular interaction between gene products (e.g. protein-protein interactions),
+      or may be a genetic interaction between genes (e.g. phenotypic suppression)
+    is_a: GeneToGeneAssociation
+    abstract: true
+    slots:
+      cross_references
```

Comment:
need '-' here in your slot definition

Reply:
Thanks for catching that, fixed on my branch (will push soon).